packages/participants-svc/src/domain/participant_agg.ts

```diff
@@ -1787,6 +1787,31 @@ export class ParticipantAggregate {
             );
         }
 
+        /*Check duplicate requests*/
+        if(existing.participantAccountsChangeRequest.find(
+            (value: IParticipantAccountChangeRequest) =>
+                value.type === accountChangeRequest.type &&
+                value.currencyCode === accountChangeRequest.currencyCode && 
+                accountChangeRequest.requestType === "ADD_ACCOUNT"
+        )){
+            throw new CannotAddDuplicateAccountError(
+                "Account change request with the same information exists already"
+            );
+        }
+
+        if(existing.participantAccountsChangeRequest.find(
+            (value: IParticipantAccountChangeRequest) =>
+                value.type === accountChangeRequest.type &&
+                value.currencyCode === accountChangeRequest.currencyCode &&
+                value.externalBankAccountId === accountChangeRequest.externalBankAccountId && 
+                value.externalBankAccountName === accountChangeRequest.externalBankAccountName &&
+                accountChangeRequest.requestType === "CHANGE_ACCOUNT_BANK_DETAILS"
```

Comment:
When receiving a create account request, the business logic should make sure that it is not accepted in any of these two cases:

- An account with that type & currency already
- A similar create account request for that type & currency already exists and is open (not approved nor rejected).